assets/css/screen.css

```diff
@@ -3034,3 +3035,65 @@ file to add some of your own styles. Well, you've come to
 the right place. Onward!
 
  */
+
+ /* custom styling */
+
+ #gh-head {
+    position: sticky;
+    top: 0;
+    background: var(--ghost-accent-color);
+ }
+
+ #gh-head nav {
+    margin: 0;
+    display: flex;
+    justify-content: flex-end;
+ }
+
+ #gh-head nav a,.gh-search,.nav-more-toggle {
+    color: #fff;
+    font-weight: 700;
+ }
+
+ .gh-burger::before,.gh-burger::after { background: #fff;}
+
+ a[href="https://content.axioned.com/"] { 
+    color: var(--color-hover);
+    opacity: 100;
+}
+
+.gh-head-button { 
+    border: 1px solid var(--color-hover);
+    background: var(--color-hover);
+}
+
+.gh-head-open #gh-head .gh-head-actions { background: transparent;}
+
+.post-card-image { aspect-ratio: 3/2;}
+
+.site-footer ,.gh-dropdown{ background: var(--ghost-accent-color);}
+
+.fluid-width-video-wrapper{
```

Comment:
why are these .fluid-width-video on here?
what's issue in the existing theme is it solving?

Reply:
This fix is for when we embed youtube video, on mobile devices the video overflows